src/cli/useInteractionReducer.ts

```diff
@@ -73,7 +73,9 @@ function backendCommitGraphToDisplayCommits(
     hasFork: boolean,
   ): DisplayCommit[] {
     const childCommits = subgraphBackendRootCommit.childCommits.map(
-      (childCommitHash) => repository.commits[childCommitHash],
+      /*  SEE - Since the commits came from the graph, childCommitHash must exist in repository.commits.
+       */
```

Comment:
Style nits:

1. imo the "SEE" isn't necessary since this doesn't seem to be particularly important
1. I think we should prefer `//`-style comments instead of `/* */` for consistency with typical JS code style, especially if the comment is just one line

```suggestion
      // Since the commits came from the graph, childCommitHash must exist in repository.commits.
```